### PR TITLE
Sig Builder

### PR DESCRIPTION
**Pull request by jmininger**

### CODE REVIEW

File: frontend/src/Frontend/Network.hs
Comment (mightybyte) [inline frontend/src/Frontend/Network.hs]: This TODO can probably be removed.

File: frontend/src/Frontend/UI/Dialogs/SigBuilder.hs
Comment (mightybyte) [inline frontend/src/Frontend/UI/Dialogs/SigBuilder.hs]: Was this meant to be uncommented?

File: frontend/src/Frontend/UI/Dialogs/SigBuilder.hs
Comment (mightybyte) [inline frontend/src/Frontend/UI/Dialogs/SigBuilder.hs:543]: Maybe, but that seems like it would present an avenue of attack.  I would probably stick with the conservative "no signatures are computed until the user has give explicit approval" philosophy.

File: frontend/src/Frontend/UI/Dialogs/SigBuilder.hs
Comment (mightybyte) [inline frontend/src/Frontend/UI/Dialogs/SigBuilder.hs]: Let's change this to say "Review"

**Review by mightybyte — APPROVED** [review]

File: frontend/src/Frontend/UI/Dialogs/SigBuilder.hs
Comment (jmininger) [reply frontend/src/Frontend/UI/Dialogs/SigBuilder.hs:543]: So the idea was still to fork after the user presses "Accept / Sign". But i wanted finer grain changes so we could present a better loading screen like: " Loading: 3/8 signatures added" -- instead of just being stuck on it as it adds all 8 sigs at once

**Pull request merged**
